requirements.txt

```diff
@@ -1,3 +1,3 @@
 chaostoolkit-lib>=0.15.0
 logzero
-slackclient
\ No newline at end of file
+slackclient==1.3.1
```

Comment:
I'm happy with that if that's the last 1.x version. Otherwise, maybe something like:

```
slackclient<2,>=1.3.1
```

Reply:
Yes, definitely.
I guess it'll be the last 1.x version but can't ensure that, let me change that to keep it clean.